scripts/deb/control

```diff
@@ -0,0 +1,8 @@
+Package: status-desktop
```

Comment:
None of these files are scripts, and should not be under `scripts`. As stated in #3367 we need something like a `resources` folder where we can put stuff like that. So it's `resources/pkg/deb/control` or something like that.